packages/ui-overlays/src/Mask/index.tsx

```diff
@@ -23,10 +23,11 @@
  */
 
 /** @jsx jsx */
-import { Component } from 'react'
+import React, { Component } from 'react'
 // @ts-expect-error ts-migrate(7016) FIXME: Could not find a declaration file for module 'no-s... Remove this comment to see the full error message
```

Comment:
you can get rid of this error by importing `"@types/no-scroll": "^2"` as a DevDependency

Reply:
Oh indeed, thanks for the tip!